cdhweb/pages/tests/test_templates.py

```diff
@@ -166,80 +174,79 @@ def test_attachments(self, client, content_page, attachment):
         # NOTE: using "img" here instead of "image" means we're
         # not actually testing the image logic; but not clear how
         # to mock or use an image object in a test
-        html = block.render(block.to_python({
-            'img': test_img, 'alternative_text': alt_text
-        }))
-        assert '<figure>' in html
+        html = block.render(
+            block.to_python({"img": test_img, "alternative_text": alt_text})
+        )
+        assert "<figure>" in html
         assert '<img srcset="' in html
         assert 'alt="picture of a kitten" ' in html
         # no caption
-        assert '<figcaption>' not in html
+        assert "<figcaption>" not in html
 
         # with caption
-        caption = 'A kitten curled up in the sunshine'
-        html = block.render(block.to_python({
-            'img': test_img, 'alternative_text': alt_text,
-            'caption': caption}))
-        assert '<figcaption>' in html
+        caption = "A kitten curled up in the sunshine"
+        html = block.render(
+            block.to_python(
+                {"img": test_img, "alternative_text": alt_text, "caption": caption}
+            )
+        )
+        assert "<figcaption>" in html
         assert caption in html
 
 
 class TestSVGImageBlock:
-
     def test_render(self):
         block = SVGImageBlock()
-        test_svg = {'url': 'graph.svg'}  # Mock(spec=Document, url='graph.svg')
-        alt_text = 'membership timeline'
-        html = block.render({
-            'image': test_svg, 'alternative_text': alt_text
-        })
-        assert ('<figure ') in html
+        test_svg = {"url": "graph.svg"}  # Mock(spec=Document, url='graph.svg')
+        alt_text = "membership timeline"
+        html = block.render({"image": test_svg, "alternative_text": alt_text})
+        assert ("<figure ") in html
         assert '<img role="img" ' in html
         # no caption, no extended description
-        assert '<figcaption>' not in html
+        assert "<figcaption>" not in html
         assert '<div class="sr-only" ' not in html
 
         # with caption & extended description
-        caption = 'membership activity from 1919 to 1942'
-        desc = 'chart shows activity in 1920 and 1940'
-        html = block.render({
-            'image': test_svg, 'alternative_text': alt_text,
-            'caption': caption, 'extended_description': desc})
-        assert ('<figcaption>%s</figcaption' % caption) in html
+        caption = "membership activity from 1919 to 1942"
+        desc = "chart shows activity in 1920 and 1940"
+        html = block.render(
+            {
+                "image": test_svg,
+                "alternative_text": alt_text,
+                "caption": caption,
+                "extended_description": desc,
+            }
+        )
+        assert ("<figcaption>%s</figcaption" % caption) in html
         assert '<div class="sr-only" id="graphsvg-desc">' in html
         assert desc in html
```

Comment:
I assume you just ended up applying black to this file?

Reply:
Yeah, sorry about all the noise...I apply it habitually to files I touch now, and forgot it wasn't on cdhweb yet!